rl_algorithms/common/networks/heads.py

```diff
@@ -93,6 +93,27 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
 
+@HEADS.register_module
+class ACERHead(nn.Module):
```

Comment:
Please use MLP instead of making head for acer. As forementioned, dividing actor and critic network might help.